raiden-ts/src/raiden.ts

```diff
@@ -194,6 +199,19 @@ export class Raiden {
 
     this.events$ = action$.pipe(filter(isActionOf(Object.values(RaidenEvents))));
 
+    this.getTokenInfo = memoize(async (token: string) => {
```

Comment:
Return-type missing, should probably be `TokenInfo`?

Reply:
We DO have type support. It's just implicit (actually explicit by the declaration of the member above, just not here). `memoize` accepts a generic type, which is a function `(token: string) => Promise<{ ...members }>`, and return this type intersected with a `MemoizedFunction` (which is just an expansion to add the `cache` member and other memoized function additional attributes), which is also of the first type and then assignable to the member. If you e.g. try to rename one of the members of the returned object, you'll see `tsc` will complain the types are incompatible, so we have stricit/full type support here.